src/inputs/dropdown/query-search-dropdown/PQuerySearchDropdown.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+    <div v-click-outside="hideMenu" class="p-query-search-dropdown">
+        <p-search :class="{'no-menu': menu ? menu.length === 0 : false}"
+                  :value="searchText"
+                  :placeholder="placeholder ? placeholder : 'Select Key: value'"
+                  disable-icon
+                  :is-focused.sync="isFocused"
+        >
+            <template #default="scope">
+                <p-tag v-for="(selectedItem, index) in proxySelected" :key="`tag-box-${index}`"
```

Comment:
It would be better to put key of `selectedItem` for a more robust defense code.


```suggestion
                <p-tag v-for="(selectedItem, index) in proxySelected" :key="`tag-box-${selectedItem.key}-${index}`"
```